modules/insticatorBidAdapter.js

```diff
@@ -22,30 +22,47 @@ config.setDefaults({
 });
 
 function getUserId() {
-  let uid;
-
-  if (storage.localStorageIsEnabled()) {
-    uid = localStorage.getItem(USER_ID_KEY);
-  } else {
-    uid = storage.getCookie(USER_ID_KEY);
+  let uid = storage.getCookie(USER_ID_KEY);
+  if (uid && isUserIdValid(uid)) {
+    const expireIn = new Date(Date.now() + USER_ID_COOKIE_EXP).toUTCString();
+    storage.setCookie(USER_ID_KEY, uid, expireIn);
```

Comment:
we 
<img width="715" alt="image" src="https://user-images.githubusercontent.com/112518464/235684400-2d24e82b-c81e-43ae-a0ff-24cc8bc7bd34.png">

Domain, path, secure etc flags that are necessary aren't set.

Reply:
![image](https://user-images.githubusercontent.com/81261035/235741149-e4f7b2c8-5d28-49d2-aded-9744d7d8acc6.png)